frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestFeatureToggleChange/ChangeRequestFeatureToggleChange.tsx

```diff
@@ -23,15 +23,15 @@ export const ChangeRequestFeatureToggleChange: FC<
         <Box
             sx={theme => ({
                 backgroundColor: Boolean(conflict)
-                    ? theme.palette.warning.light
+                    ? theme.palette.neutral.light
```

Comment:
same color but semantic name as it wasn't a warning but neutral header background